package.json

```diff
@@ -90,7 +90,7 @@
     "react": ">= 16.8"
   },
   "dependencies": {
-    "attr-accept": "^2.2.1",
+    "attr-accept": "^2.2.2",
```

Comment:
The `^` should catch patches as far as I know.

Reply:
Indeed it seems to be updating to 2.2.2.
I guess this change is not required.